keel-web/src/test/kotlin/com/netflix/spinnaker/keel/integration/SpringStartupTests.kt

```diff
@@ -16,14 +16,7 @@ import org.springframework.test.context.junit.jupiter.SpringExtension
 @ExtendWith(SpringExtension::class)
 @SpringBootTest(
   classes = [KeelApplication::class],
-  webEnvironment = MOCK,
-  properties = [
-    "sql.enabled=true",
-    "keel.plugins.ec2.enabled=true",
-    "sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename",
-    "sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename",
-    "spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver"
-  ]
```

Comment:
All these properties are set in `src/test/resources/application.properties` now so this was redundant